salt/modules/mssql.py

```diff
@@ -0,0 +1,346 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide MS SQL Server compatibility to salt.
+
+:depends:   - FreeTDS
+            - pymssql Python module
+
+:configuration: In order to connect to MS SQL Server, certain configuration is
+    required in minion configs/pillars on the relevant minions. Some sample
+    pillars might look like::
+
+        mssql.server: 'localhost'
+        mssql.port:   1433
+        mssql.user:   'sysdba'
+        mssql.password:   'Some preferable complex password'
+        mssql.database: ''
+
+    The default for the port is '1433' and for the database is '' (empty string);
+    in most cases they can be left at the default setting.
+    Options that are directly passed into functions will overwrite options from
+    configs or pillars.
+'''
+
+# Import python libs
+from json import JSONEncoder, loads
```

Comment:
`from __future__ import absolute_import` - that should help clean up that pylint error.